website/docs/docs/cloud/connect-data-platform/connect-amazon-athena.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Connect Amazon Athena"
+id: connect-amazon-athena
+description: "Configure Amazon Athena connection."
+sidebar_label: "Connect Amazon Athena"
+---
+
+# Connect Amazon Athena <Lifecycle status="beta" />
+
+:::note beta
+
+This is a beta feature with limited availability. A public preview will follow shortly, for wider early access. For more information, check out our [product lifecycle](/docs/dbt-versions/product-lifecycles#dbt-cloud) page. 
+
+:::
+
+Your environment(s) must be on ["Keep on latest version"](/docs/dbt-versions/versionless-cloud) to use the Amazon Athena connection.
```

Comment:
```suggestion
Your environment(s) must be on [Keep on latest version](/docs/dbt-versions/versionless-cloud) to use the Amazon Athena connection.
```

Reply:
We've been using the quotes elsewhere, so I'm gonna leave it to remain consistent